docs/guides/search-path.md

```diff
@@ -7,18 +7,20 @@ sidebar_label: Using search_path
 # Using search_path to target connections and aggregators
 
 You are probably here for one of the following reasons:
-- You cant figure out why steampipe isn't using your aggregator
-- You want to run steampipe query, check, or dashboard against a specific connection
-- You want to change your 'default' connection
-- You've seen references to the search path elsewhere, but you're not sure why its important
-- You asked what you thought was a simple question on the steampipe slack, and instead of an answer they sent you this link (ugh...homework...)
+- You can't figure out why Steampipe isn't using your aggregator
+- You want to run `steampipe query`, `check`, or `dashboard` against a specific connection
+- You want to change your default connection
+- You've seen references to the search path elsewhere, but you're not sure why it's important
+- You asked what you thought was a simple question on the Steampipe Slack, and instead of an answer they sent you this link (ugh...homework...)
 
-This guide will attempt to answer these questions, in 5 minutes or less.
+This guide answers your questions.
```

Comment:
I prefer the original text

Reply:
How about "answers your questions in 5 minutes or less"?

("will attempt to" is a weaker construction)

Reply:
Its intentionally weak - i don't know if it will answer their questions or not.  I'm not going to die on this hill though.

Reply:
😂  Sorry, the copy-editor region of my brain has gotten strongly activated lately, and I may be overreaching in some cases.